packages/lit-ssr/src/lib/render-lit-html.ts

```diff
@@ -464,6 +490,7 @@ export function* renderValue(
   value: unknown,
   renderInfo: RenderInfo
 ): IterableIterator<string> {
+  patchIfDirective(value);
```

Comment:
Is this where nested directive results would be patched?

Reply:
No, it's on the return value of `render` in the patch: https://github.com/Polymer/lit-html/pull/1599/commits/09d6df9a9563024d70a35ab6a9705df9f4e08dee#diff-b8cfd0fdb8410f04f051c052e6ce548e5403f407d6954d48df02b7c5bdb0e506R89